src/obs-plugin/plugin-main.mm

```diff
@@ -108,6 +113,26 @@ bool obs_module_load(void)
             action->setText(obs_module_text("Start Virtual Camera"));
             obs_output_stop(output);
         } else {
+            
+            obs_video_info ovi;
+            obs_get_video_info(&ovi);
+            if(getAspectRatio(ovi.base_width, ovi.base_height) != getAspectRatio(ovi.output_width, ovi.output_height)) {
+                QMessageBox msgBox;
+                msgBox.setText(QString::fromStdString(obs_module_text("Warning: Aspect ratios don't match")));
```

Comment:
I guess my core question here is whether this is really the responsibility of our plugin to flag this to the user. Any other output from OBS (e.g. rtmp, decklink) is going to have the same problem since OBS allows different canvas and output resolutions.

So maybe it comes down to whether we think this will sufficiently lower our support load. If that is the case it might be worth doing anyways. But I think most of the resolution issues raised have referred to the garbled output which should be fixed in #144

What do you think?

Reply:
This is a good question!

While it is correct that OBS and other plugins don't do that either, I think that people who normally use OBS have more tech-expertise and will know not to change that or will know what is wrong. As this is also targeted at people that have no idea of this, personally I think that it would be a good thing for many that would just wonder what is wrong and maybe not even open a support request because they think something with their system is broken.
On the other hand, this isn't specified in the RFC, so adding it would be a step in the wrong direction. Also most garbled output is indeed connected to #144.
I think that this is a point that maybe OBS should tackle in general, but that is out of scope now 😄 

Maybe hear what @dodgepong has to say?

Reply:
If mismatched aspect ratios are not causing a problem for this plugin anymore, I would remove this. If they are still causing a problem, then I would make a proposal for addressing this in core. Either way, I agree this is probably out of scope for this plugin.

Reply:
I feel like addressing this in core would be a good idea regardless of this plugin, but you're right. 
Going to close this.